content/modules/redisearch/search-json-quickstart.md

```diff
@@ -0,0 +1,310 @@
+---
+Title: Search JSON quick start
+linkTitle: Quick start (with JSON)
+description: RediSearch quick start on how to index JSON documents and run search queries.
+weight: 22
+alwaysopen: false
+categories: ["Modules"]
+module: RediSearch
+aliases: /modules/redisearch/search-json-quickstart/
+---
+
+This quick start shows you how to index documents stored as JSON and run search queries against the index.
+
+## Prerequisites
+
+For this quick start tutorial, you need:
+
+- Either:
+
+    - A [Redis Cloud]({{<relref "/modules/modules-quickstart.md">}}) database with [Redis Stack]({{<relref "/modules/redis-stack">}})
+
+    - A [Redis Enterprise Software]({{<relref "/modules/install/add-module-to-database">}}) database with RediSearch (v2.2 or later) and RedisJSON (v2.0 or later)
+
```

Comment:
Add some indicator here that the two below are not part of the "either" list. Something like AND?